src/ReactiveSocket.cpp

```diff
@@ -90,6 +90,12 @@ void ReactiveSocket::requestFireAndForget(Payload request) {
   connection_->onNextFrame(frame);
 }
 
+void ReactiveSocket::metadataPush(Payload metadata) {
+  FrameMetadata metadata_ = FrameMetadata(std::move(metadata));
```

Comment:
local variables should not have the trailing _. This is for class member variables.